examples/format_example_fabric_intermediary.modip.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../json_schema/1.0.0.schema.json",
+  "_comment": [
+    "The '_comment' field is non-standard and only exists for explanation purposes",
+    "This file demonstrates an example classpath library in the MODIP Format",
+    "It is based off the Fabric Intermediary"
+  ],
+  "schemaVersion": "1.0.0",
+  "id": "framework-fabric-intermediary",
+  "name": "Fabric Intermediary",
+  "summary": "Short summary about Fabric Intermediary.",
+  "description": "Long description about Fabric Intermediary.",
+  "media": [
+    {
+      "rel": "icon",
+      "type": "image",
+      "src": "https://example.com/fabric-icon.png",
+      "sha256": "< sum goes here >"
+    }
+  ],
+  "authors": [
+    {
+      "team": true,
+      "id": "111111-111111-111111-111111-111111",
+      "name": "Fabric Contributors",
+      "uri": "https://github.com/fabricmc",
+      "members": [
+        {
+          "id": "111111-111111-111111-111111-111111",
+          "name": "Example Fabric Contributor",
+          "uri": "https://example.com/example-webpage"
+        }
+      ]
+    }
+  ],
+  "tags": [
+    "framework"
+  ],
+  "license": {
+    "id": "CC0-1.0",
+    "name": "CC0 1.0 Universal",
+    "uri": "https://creativecommons.org/publicdomain/zero/1.0/",
+    "modpackUsage": "alwaysAllow"
+  },
+  "links": [
+    {
+      "rel": "scm",
+      "name": "Fabric Intermediary GitHub",
+      "uri": "https://github.com/fabricmc/intermediary"
+    }
+  ],
+  "versions": [
+    {
+      "id": "1.16.1",
+      "name": "Intermediary 1.16.1",
+      "releaseDate": "2020-08-05T20:59:00Z",
+      "changelog": "Intermediary changelog goes here...",
+      "installation": {
+        "method": "classpathLibrary",
+        "versionJson": {
+          "name": "net.fabricmc:intermediary:1.16.1",
+          "url": "https://maven.fabricmc.net"
```

Comment:
Hmm, is this correct? It seems very strange if it's the intermediary that creates the version JSON. Wouldn't the intermediary add to the classpath, but fabric loader contributes the main class? There may be several mods that contribute independently to version.json especially the library dependencies.

Reply:
The intermediary would add to the classpath, yes. The `versionJson` field is how the library would be described if it were placed in a version json, under `libraries`. It isn’t the entire version json. A launcher could either add the contents of `versionJson` to `libraries`, or it could simply parse it, download it, and add to the classpath. The field may need to be renamed though, because `versionJson` suggests the full thing. I will clean up the definition.

Reply:
resolved